doc/orchestrator.rst

```diff
@@ -63,108 +326,207 @@ on that compute host.
+- The Experiment Driver Script
+   The experiment driver script launches and manages
+   the example entities with the ``Experiment`` API.
+   In the driver script, we use the ``Experiment``
+   to create and launch a colocated ``Model``.
 
-.. currentmodule:: smartsim.entity.model
+The Application Script
+======================
+To begin writing the application script, provide the imports:
 
-.. automethod:: Model.colocate_db_tcp
-    :noindex:
+.. code-block:: python
 
-.. automethod:: Model.colocate_db_uds
-    :noindex:
+  from smartredis import ConfigOptions, Client, log_data
+  from smartredis import *
+  import numpy as np
 
-Here is an example of creating a simple model that is colocated with an
-``Orchestrator`` deployment using Unix Domain Sockets
+Initialize the Clients
+----------------------
+To establish a connection with the colocated database,
+initialize a new SmartRedis client and specify `cluster=False`
+since our database is single-sharded:
 
 .. code-block:: python
 
-  from smartsim import Experiment
-  exp = Experiment("colo-test", launcher="auto")
+  # Initialize a Client
+  colo_client = Client(cluster=False)
+
+.. note::
+    Since there is only one database launched in the Experiment
+    (the colocated database), specifying a a database address
+    is not required when initializing the client.
+    SmartRedis will handle the connection.
+
+.. note::
+   To create a client connection to the colocated database, the colocated Model must be launched
+   from within the driver script. You must execute the Python driver script, otherwise, there will
+   be no database to connect the client to.
+
+Store Data
+----------
+Next, using the SmartRedis client instance, we create and store a NumPy tensor using
+``Client.put_tensor()``:
+
+.. code-block:: python
+
+    # Create NumPy array
+    array_1 = np.array([1, 2, 3, 4])
+    # Store the NumPy tensor
+    colo_client.put_tensor("tensor_1", array_1)
+
+Retrieve Data
+-------------
+Next, retrieve the tensor using ``Client.get_tensor()``:
+
+.. code-block:: python
 
-  colo_settings = exp.create_run_settings(exe="./some_mpi_app")
+    # Retrieve tensor from driver script
+    value_1 = colo_client.get_tensor("tensor_1")
+    # Log tensor
+    colo_client.log_data(LLInfo, f"The colocated db tensor is: {value_1}")
 
-  colo_model = exp.create_model("colocated_model", colo_settings)
-  colo_model.colocate_db_uds(
-          db_cpus=1,              # cpus given to the database on each node
-          debug=False             # include debug information (will be slower)
-          ifname=network_interface # specify network interface(s) to use (i.e. "ib0" or ["ib0", "lo"])
-  )
-  exp.start(colo_model)
+When the Experiment completes, you can find the following log message in `colo_model.out`::
+    Default@21-48-01:The colocated db tensor is: [1 2 3 4]
 
+The Experiment Driver Script
+============================
+To run the application, specify a Model workload from
```

Comment:
To run the application, specify a Model workload from
within the workflow (Experiment).
Defining workflow stages requires the utilization of functions associated
with the ``Experiment`` object.
In this example, we instantiate an ``Experiment`` object with the name ``getting-started``.
We setup the SmartSim ``logger`` to output information from the Experiment.

->

To run the application, a ``Model`` object must be configured and launched via the ``Experiment`` driver script.  Defining the workflow entities requires the utilization of the ``Experiment`` object API.
In this example, we instantiate an ``Experiment`` object with the name ``getting-started``, and we setup the SmartSim ``logger`` to output information from the Experiment.